static/static/csv/GeoNames-Monthly-Diff-Report-March-2023.csv

```diff
@@ -0,0 +1,70 @@
+geoname_id,3456058,6317783,Novo Horizonte,Novo Horizonte,Brazil,Brazil,BR,BR,SC,SC
+geoname_id,8602689,3385745,Uniao,Uniao,Brazil,Brazil,BR,BR,PI,PI
+geoname_id,3387423,6318704,Sitio Novo do Tocantins,Sitio Novo do Tocantins,Brazil,Brazil,BR,BR,TO,TO
+geoname_id,6318649,6323785,Selviria,Selviria,Brazil,Brazil,BR,BR,MS,MS
+geoname_id,6323812,6316760,Canarana,Canarana,Brazil,Brazil,BR,BR,MT,MT
+geoname_id,3472007,6316401,Antonio Joao,Antonio Joao,Brazil,Brazil,BR,BR,MS,MS
+geoname_id,3405806,6319378,Belterra,Belterra,Brazil,Brazil,BR,BR,PA,PA
+geoname_id,6324176,3663495,Manoel Urbano,Manoel Urbano,Brazil,Brazil,BR,BR,AC,AC
+geoname_id,6318418,3449165,Sao Francisco,Sao Francisco,Brazil,Brazil,BR,BR,SP,SP
+geoname_id,6321040,3449304,Sao Desiderio,Sao Desiderio,Brazil,Brazil,BR,BR,BA,BA
+geoname_id,6319281,3664464,Cruzeiro do Sul,Cruzeiro do Sul,Brazil,Brazil,BR,BR,AC,AC
+geoname_id,3668332,3668333,Santo Tomas,Santo Tomas,Colombia,Colombia,CO,CO,ATL,ATL
+geoname_id,2619860,2619859,Horning,Horning,Denmark,Denmark,DK,DK,82,82
+geoname_id,3492553,3492552,Tabique,Tabique,Dominican Republic,Dominican Republic,DO,DO,5,5
+geoname_id,3497281,3497280,Magante,Magante,Dominican Republic,Dominican Republic,DO,DO,9,9
+geoname_id,590977,590975,Kunda,Kunda,Estonia,Estonia,EE,EE,"60, 903","60, 903"
+geoname_id,2957733,3206785,Altendorf,Altendorf,Germany,Germany,DE,DE,BY,BY
+geoname_id,2906334,2906333,Hennickendorf,Hennickendorf,Germany,Germany,DE,DE,BB,BB
+geoname_id,3206177,11608792,Grafenberg,Grafenberg,Germany,Germany,DE,DE,BW,BW
+geoname_id,3728088,3728087,Cazeau,Cazeau,Haiti,Haiti,HT,HT,OU,OU
+geoname_id,3723783,3723782,Jackson,Jackson,Haiti,Haiti,HT,HT,SE,SE
+geoname_id,3607516,3607491,La Lima,La Lima,Honduras,Honduras,HN,HN,LE,LE
+geoname_id,3607965,3607960,La Esperanza,La Esperanza,Honduras,Honduras,HN,HN,FM,FM
+geoname_id,1936351,1630923,Polokarto,Polokarto,Indonesia,Indonesia,ID,ID,JT,JT
+geoname_id,8379755,1625360,Tamanan,Tamanan,Indonesia,Indonesia,ID,ID,JI,JI
+geoname_id,12032400,7087267,Subagan,Subagan,Indonesia,Indonesia,ID,ID,BA,BA
+geoname_id,1647575,1647574,Buol,Buol,Indonesia,Indonesia,ID,ID,ST,ST
+geoname_id,3180511,8980123,Carignano,Carignano,Italy,Italy,IT,IT,"21, TO","21, TO"
+geoname_id,1855886,11821857,Nakamura,Nakamura,Japan,Japan,JP,JP,35,35
+geoname_id,1735339,1735297,Mukah,Mukah,Malaysia,Malaysia,MY,MY,13,13
+geoname_id,1291452,1291451,Thingangyun,Thingangyun,Myanmar,Myanmar,MM,MM,6,6
+geoname_id,11696088,3132911,Vale,Vale,Norway,Norway,NO,NO,38,38
+geoname_id,1719684,1719683,Camiling,Camiling,Philippines,Philippines,PH,PH,"03, TAR","03, TAR"
+geoname_id,11428151,983038,Letaba,Letaba,South Africa,South Africa,ZA,ZA,LP,LP
+geoname_id,2514410,2514411,Los Villares,Los Villares,Spain,Spain,ES,ES,"AN, J","AN, J"
+geoname_id,1616797,1616794,Ban Mai,Ban Mai,Thailand,Thailand,TH,TH,14,14
+subdivision_codes,2198148,2198148,Suva,Suva,Fiji,Fiji,FJ,FJ,C,"C, 12"
+subdivision_codes,2198148,2198148,Suva,Suva,Fiji,Fiji,FJ,FJ,C,"C, 12"
```

Comment:
Is this duplicate intentional?

Reply:
oh, thank you dallas. i forgot to check for duplicates.